newsletters/2024-03.md

```diff
@@ -0,0 +1,70 @@
+---
+tags: obf
+---
+
+_Having trouble viewing this email or looking for a permalink? [View this newsletter in your browser](https://github.com/OBF/newsletter/blob/master/newsletters/2024-03.md)._
+
+_This email was sent to the [bosc-announce](https://groups.google.com/g/bosc-announce) and [open-bio-l](http://mailman.open-bio.org/mailman/listinfo/open-bio-l/) mailing lists._
+
+# OBF Newsletter March 2024
+
+TL;DR: In this newsletter we provide key dates for BOSC 2024; share the upcoming OBF event fellowship, highlight an upcoming ISCBacademy event, and share upcoming community opportunities.
+
+In longer form: 
+
+## OBF News
+
+### BOSC news
+
+The 25th(!) annual Bioinformatics Open Source Conference (BOSC 2024) will take place on July 15/16 as part of ISMB 2024 in Montréal, Canada, and online. As in prior years, there will be a _CollaborationFest_, a collaborative work event (including but not limited to hacking) happening around BOSC, this year right after the event on July 17/18.
+
+There is still time to submit your abstracts for posters and talks. And there [is even an early poster submission/acceptance route that you can use until March 21](https://www.open-bio.org/2024/02/18/early-poster-acceptance-2024/). With this we want to make it easier for people to support them in their travel planning.
+
+The full list of key submission/notification dates is: 
+
+* March 21, 2024: [Early Poster Acceptance submission deadline](https://www.open-bio.org/2023/03/09/bosc-early-poster-acceptance/)
+* March 29: Early Poster Acceptance notification
+* April 19: Regular abstract submission deadline (talks and posters)
+* May 13: Talk/poster acceptance notification
+* May 20: Late poster submission deadline
+
+The actual conference dates are:
+
+* July 12-16, 2024: ISMB 2024 (Montréal, Canada and online)
+* July 15-16, 2024: BOSC 2024 (part of ISMB)
+* July 17-18: Post-BOSC CollaborationFest
+
+
```

Comment:
To keep up with BOSC news, please follow us on [LinkedIn](https://www.linkedin.com/groups/14344023/), [Bluesky](https://bsky.app/profile/bosc.bsky.social), [Mastodon](https://genomic.social/@BOSC), and [Slack](https://join.slack.com/t/obf-bosc/shared_invite/zt-n5ur1gsj-z2C~69_4lYTFPg5tbWA8Ew)! (We are no longer on Twitter/X; [find out why](https://www.open-bio.org/2023/11/20/leaving-x/).)

Reply:
(I meant to add that as a new line; not sure if that worked)

Reply:
no worries, i added it manually!